### DISCUSSION TITLE
关于搜索屏蔽加密文件夹下文件

### DISCUSSION THREAD

**Discussion by huarzone** [spencerwooo/onedrive-vercel-index#618] [Ideas]
现在的搜索功能能否屏蔽一下路径属于加密文件路径下的文件，只有输入密码后才可以在搜索时显示，🧐就算前端防的不是很安全（防君子不防小人），最起码有这么个功能可以避免掉大部分人的搜索。

**Comment by spencerwooo** [comment]
这个也讨论过：https://github.com/spencerwooo/onedrive-vercel-index/pull/283#discussion_r790121386
